packages/effects-core/src/comp-vfx-item.ts

```diff
@@ -368,6 +370,83 @@ export class CompVFXItem extends VFXItem<void | CalculateItem> {
     return res;
   }
 
+  hitTest (ray: Ray, x: number, y: number, regions: Region[], force?: boolean, options?: CompositionHitTestOptions): Region[] {
```

Comment:
Is this moving the relevant methods from the Composition class to the ComVFXItem class?

Reply:
yes